addons/account/models/company.py

```diff
@@ -273,7 +273,7 @@ class ResCompany(models.Model):
     def get_next_batch_payment_communication(self):
         '''
         When in need of a batch payment communication reference (several invoices paid at the same time)
-        use batch_payment_sequence_id to get it (eventually create it first): e.g BATCH/2024/00001
+        use batch_payment_sequence_id to get it (eventually create it first): e.g PAY/2024/00001
```

Comment:
Nice catch ! you could also change the "When in need of a batch payment" since it's a group payment but it's a nitpicking :smile: